scripts/seed.ts

```diff
@@ -1,32 +1,114 @@
 import { db } from "../src/db";
-import { usersTable } from "../src/db/schema";
+import { eventsTable, userEmbeddingsTable, usersTable } from "../src/db/schema";
+
+// Helper to generate mock 128-dimensional vector
+function generateMockEmbedding(): number[] {
+  return Array.from({ length: 128 }, () => Math.random() * 2 - 1);
```

Comment:
```suggestion
  return Array.from({ length: 1024 }, () => Math.random() * 2 - 1);
```

This is an incoming change from #19 , the embedding dimensions will be **1024** instead of 128 dimensions. I've updated the DBML in that PR, but in the meantime, please change all instances of 128 to 1024.